ui/pages/swaps/fee-card/fee-card.js

```diff
@@ -167,9 +166,7 @@ export default function FeeCard({
                   onClick={onQuotesClick}
                   className="fee-card__quote-link-text"
                 >
-                  {isBestQuote
```

Comment:
Any context behind this change we should know? Is this unreliable data or an unnecessary distinction?